packages/block-library/src/list/transforms.js

```diff
@@ -77,12 +77,35 @@ const transforms = {
 				ul: listContentSchema.ul,
 			},
 			transform( node ) {
-				return createBlock( 'core/list', {
-					...getBlockAttributes(
-						'core/list',
-						node.outerHTML
-					),
+				const attributes = {
 					ordered: node.nodeName === 'OL',
+				};
+
+				if ( attributes.ordered ) {
+					const type = node.getAttribute( 'type' );
+
+					if ( type ) {
+						attributes.type = type;
+					}
+
+					if ( node.getAttribute( 'reversed' ) !== null ) {
```

Comment:
If `reversed` was explicitly set to `false`, this will still reverse the list, right?

While I understand that doing something like that does not make a lot of sense, it is still possible that some component simply passes on a Boolean prop.

What about this?

```suggestion
					const reversed = node.getAttribute( 'reversed' );
					if ( reversed !== null && reversed !== 'false' ) {
```

Reply:
You're right, the attribute value doesn't matter.

Reply:
But I don't get what you're proposing... I check for the mere presence of the attribute, which seems correct to me. You suggest to check that it is not `false`, which is contrary to what you say:

> While I understand that doing something like that does not make a lot of sense, it is still possible that some component simply passes on a Boolean prop.

Reply:
```
<ol reversed="false">
    <li>Item 1</li>
    <li>Item 2</li>
</ol>
```

This will render like so:

```
2. Item 1
1. Item 2
```

I think this is wrong, or, at least, unexpected. My committable suggestion should fix that, no?

Reply:
I disagree. For paste, we should look preserve what is rendered. If a list was reversed, even with an unexpected value, it should stay reversed.

Reply:
@tfrommen Does that make sense? We shouldn't alter the code to the rendered result is different. The user doesn't really care what the underlying code is, only that the pasted content looks the same.